### ISSUE TITLE
 Error Mathjax.tpl

### ISSUE THREAD

**Issue by Shoaibashk** [danielfrg/mkdocs-jupyter#16]
notes$ mkdocs serve
```Bash
INFO    -  Building documentation... 
INFO    -  Cleaning site directory 
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - MCN/Practicals/MCN-EXP-1.ipynb
  - MCN/Practicals/MCN-EXP-2.ipynb
  - MCN/Unit-1/index.md 
ERROR   -  Error reading page 'MCN/Practicals/MCN-EXP-1.ipynb': mathjax.tpl 

mathjax.tpl
```

I am using latest version of **mkdocs-jupyter==0.13.0**
Unable to find solution.

**Comment by Sayam753** [comment]
Hi @Shoaibashk 
I also got this error sometime back. And I resolved it by reverting `nbconvert` to `5.6.1`.
So, can you check if `mkdocs serve` works good after `pip install nbconvert==5.6.1`?

**Comment by Shoaibashk** [comment]
Yes, **It's Working.** 
works great ```pip install nbconvert==5.6.1```
Thanks A lot @Sayam753
